mlflow/server/js/src/components/ExperimentViewUtil.js

```diff
@@ -359,11 +359,11 @@ export default class ExperimentViewUtil {
       }
     });
     const parentRows = [...Array(runInfos.length).keys()].flatMap((idx) => {
-      if (treeNodes[idx].isCycle() || !treeNodes[idx].isRoot()) return [];
+      if (!treeNodes[idx].isCycle() && !treeNodes[idx].isRoot()) return [];
```

Comment:
Include runs that are part of a parent-run cycle (e.g. run A's parent is run B, and run B's parent is A) in the list of runs shown in the UI, but don't allow expanding their child runs - this seems preferable to hiding them entirely and simplifies keeping track of the index of a run when its checkbox is clicked.